v2/Makefile

```diff
@@ -13,6 +13,9 @@ templates_lua = lua/openresty_org/templates.lua
 lua_files := $(sort $(filter-out $(i18n_lua) $(templates_lua),$(wildcard lua/*.lua lua/openresty_org/*.lua)))
 tt2_files := $(sort $(wildcard templates/*.tt2))
 
+css_file = css/main.css
+less_file = css/main.less
```

Comment:
Better use gmake functions `$(wildcard ...)` and `$(patsubst ...)` to generate the css and less file list automatically?

Reply:
I changed this a little bit. Currently not all less files need the compilation, only need `main.css` on the website, so I guess it's enough for now?

Reply:
@bambooom It's fine.